automated/linux/torizon/ostree-upgrade-tests.sh

```diff
@@ -5,7 +5,7 @@ set -x
 
 ostree remote add $OSTREE_REMOTE_NAME $OSTREE_URL || lava-test-case ostree-upgrade-remote-add --result fail
 
-SORTED_VERSIONS=$(ostree remote refs lavacloud | grep laa-qemu/ | awk -F'/' '{
+SORTED_VERSIONS=$(ostree remote refs lavacloud | grep $OSTREE_REF/ | awk -F'/' '{
```

Comment:
shouldn't we replace lavacloud too with $OSTREE_REMOTE_NAME?

Reply:
yes